lilcom/lilcom_c_extension.c

```diff
@@ -260,7 +260,7 @@ int decompress_int16_internal(int num_axes, int axis,
                      was noticed in this function.
      """
  */
-static PyObject * decompress_int16(PyObject * args, PyObject * keywds)
+static PyObject * decompress_int16(PyObject * self, PyObject * args, PyObject * keywds)
```

Comment:
urrgh... I think you may have been doing this with python2.  I wanted to make this  a python3 project, as python2 is deprecated now.  Please clarify in the README, so no-one makes this mistake in future; and see if there's a way in the setup.py to require python3

Reply:
I've been doing this with python3. The code doesn't compile with python2. Should I remove the "self" variable?

Reply:
Are you sure?  Because I thought I removed the `self` to get this to compile with Python3.
See here
 https://docs.python.org/3/extending/extending.html

Reply:
Were you able to compile without self?
The link you pointed also uses self I think. BTW, I have a working version of the "spam" example in this link at the python_interface branch.